docs/platform/concepts/service_backups.rst

```diff
@@ -6,12 +6,14 @@ This article provides information on general rules for handling service backups
 About backups at Aiven
 ----------------------
 
-All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. Backups at Aiven are stored in the object storage of the cloud region where a service runs (for example, S3 for AWS or GCS for GCP). You can check the location of your service's backups in `Aiven Console <https://console.aiven.io/>`_ > your service's homepage > **Backups**.
+All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. Backups at Aiven are stored in the object storage of the cloud region where the service is first created (for example, S3 for AWS or GCS for GCP). You can check the location of your service's backups in `Aiven Console <https://console.aiven.io/>`_ > your service's homepage > **Backups**.
 
 The backup retention times vary based on the service and the selected service plan. 
 
 Aiven takes service backups for managing purposes. These backups are compressed and encrypted by the Aiven management platform and, as such, are not available for download for any service type.
 
+.. note::  In case of a change of cloud provider or availability zone for a service, its backups are not migrated with it.
```

Comment:
Can we please get rid of this "in case of", which is impersonal and mostly for formal notices and announcements. How about "If you change a cloud provider or an availability zone for your service, its backups are not migrated from their original location."

Reply:
Ack!